src/views/Predictions/helpers.ts

```diff
@@ -1,15 +1,44 @@
-import { ethers } from 'ethers'
+import { BigNumber, ethers } from 'ethers'
 import { NodeRound } from 'state/types'
 import { formatBigNumberToFixed } from 'utils/formatBalance'
 import getTimePeriods from 'utils/getTimePeriods'
 
-export const formatUsdv2 = (usd: ethers.BigNumber) => {
-  return `$${formatBigNumberToFixed(usd, 3, 8)}`
+const MIN_PRICE_USD_DISPLAYED = BigNumber.from(100000)
+const MIN_PRICE_BNB_DISPLAYED = BigNumber.from('1000000000000000')
+const DISPLAYED_DECIMALS = 3
+
+type formatPriceDifferenceProps = {
+  price?: BigNumber
+  minPriceDisplayed: BigNumber
+  unitPrefix: string
+  decimals: number
+}
+
+const formatPriceDifference = ({
+  price = BigNumber.from(0),
+  minPriceDisplayed,
+  unitPrefix,
+  decimals,
+}: formatPriceDifferenceProps) => {
+  const value = price
+  let prefix = unitPrefix
+  let priceToFormat = value
+  const sign = value.isNegative() ? BigNumber.from(-1) : BigNumber.from(1)
+
+  if (value.abs().lt(minPriceDisplayed)) {
+    prefix = `<${unitPrefix}`
+    priceToFormat = minPriceDisplayed.mul(sign)
+  }
```

Comment:
Totally valid code but try and avoid using `let` and return ``<${unitPrefix}`` early.

Reply:
I also like to avoid `let` as much as possible, just wanted to mutualize the condition here.
I will use an early return instead.